css/styles.css

```
@@ -1,122 +1,238 @@
-  text-decoration: none;
-}
-
-.green-button {
-  -webkit-border-radius: 26;
-  -moz-border-radius: 26;
-  border-radius: 26px;
-  font-family: Georgia;
-  color: #ffffff;
-  font-size: 45px;
-  background: #34d934;
-  padding: 15px 25px 15px 25px;
-  text-decoration: none;
-  margin-left: 5px;
-  position: relative;
-  left: 50px;
-}
-
-.green-button:hover {
-  background: #3cfa58;
-  text-decoration: none;
-}
-
-/********************* FOOTER ****************/
-
-footer {
-	margin-top: 50px;
-}
-
- p { 
- 	margin-top: 10px;
-	text-align: center;
-}
-
-.refButton {
-  -webkit-border-radius: 26;
-  -moz-border-radius: 26;
-  border-radius: 26px;
-  font-family: Georgia;
-  color: black;
-  font-size: 40px;
-  background: yellow;
-  padding: 15px 25px 15px 25px;
-  text-decoration: none;
-  display: block;
-  margin-top: 50px;
-  margin-left: auto;
-  margin-right: auto
-}
+body{
+  background-color: whitesmoke;
+  text-align: center;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+/********************* CLASS SELECTORS ****************/
+.name-of-game {
+  text-align: center;
+  font-size: 60px;
+  font-family: cursive;
+  margin-top: 1%;
+  margin-bottom: 1%;
+  text-shadow: 3px 0 #DA0463;
+}
+
+.game-conteiner {
+	text-align: center;
+}
+
+.game-task {
+	text-align: center;
+	font-size: 32px;
+	font-family: Georgia;
+}
+
+.game-response {
+  text-align: center;
+  font-size: 25px;
+  font-family: Georgia;
+}
+
+.description {
+  font-family: Georgia;
+  font-size: 25px;
+  margin-bottom: -5px;
+}
+
+.link-description {
+  font-family: Georgia;
+  font-style: italic;
+  font-size: 20px;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+/********************* BUTTONS ****************/
+.color-button {
+  font-size: 3rem;
+  display: none;
+  margin: 10px;
+  margin-top: 0px;
+  margin-bottom: 45px;
+  max-width: 200px;
+  width: 25%;
+}
+
+.start-again-button {
+  border: 4px solid #404B69;
+  font-size: 2rem;
+  color: #DA0463;
+  margin: 10px;
+  margin-top: 0px;
+  margin-bottom: 32px;
+  display: none;
+  padding: 10px 15px 10px 15px;
+}
+
+.start-button {
+  color: black;
+  display: inline-block;
+  margin-top: 50px;
+  margin-bottom: 30px;
+  font-size: 3rem;
+  width: 300px;
+}
+
+.btn {
+  outline: none;
+  border: 4px solid grey;
+  font-family: 'Georgia', sans-serif;
+  line-height: 2;
+  font-weight: 900;
+  border-radius: 15px;
+  min-height: 100px;
+  text-align: center;
+  background-color: white;
+  cursor: pointer;
+  
+  
+}
+
+.btn:hover {
+  background: #EEEEEE;
+  border: 4px solid black;
+  cursor: pointer;
+  text-shadow: 2px 0 #DBEDF3;
+}
+/********************* FOOTER ****************/
+footer {
+  font-family: sans-serif;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 5rem;
+}
+
+ footer p { 
+	text-align: center;
+}
+
+/********************* HIDE ****************/
+.hide {
+  visibility: hidden;
+}
+
+/**** Name of Buttons with first-letter uppercase ****/
+button::first-letter {

```

better to set this property for container with buttons.
```suggestion
.buttons-container button::first-letter {

```

---

I did, but with class ".game-container"